proto-lens-protoc/src/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -71,11 +72,11 @@ data UseReexport = UseReexport | UseOriginal
 generateModule :: ModuleName
                -> [ModuleName]  -- ^ The imported modules
                -> SyntaxType
-               -> UseReexport
+               -> (ImportDecl -> ImportDecl)
```

Comment:
A type synonym would still be nice for discoverability. You already found a name for it: ModifyImports.